docs/source/daml2ts/index.rst

```diff
@@ -0,0 +1,290 @@
+.. Copyright (c) 2020 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+TypeScript!
```

Comment:
I understand you're excited but I'm not sure that makes a lot of sense to our users. I suggest something like
> `daml2ts` - A TypeScript codegen for DAML